src/axom/primal/operators/detail/evaluate_integral_impl.hpp

```diff
@@ -60,30 +96,39 @@ inline double evaluate_scalar_line_integral_component(const primal::BezierCurve<
 /*!
  * \brief Evaluate a vector field line integral on a single Bezier curve.
  *
- * Evaluate the vector field line integral with MFEM integration rule
+ * Evaluate the vector field line integral with MFEM integration rule if available, 
+ *  or axom/numerics/quadrature otherwise, which are less robust for large npts (> ~100)
  *
  * \param [in] c the Bezier curve object
  * \param [in] integrand the lambda function representing the integrand. 
  * Must accept a 2D point as input and return a 2D axom vector object
- * \param [in] quad the mfem integration rule containing nodes and weights
+ * \param [in] npts The number of quadrature points in the rule
  * \return the value of the integral
  */
 template <class Lambda, typename T, int NDIMS>
 inline double evaluate_vector_line_integral_component(const primal::BezierCurve<T, NDIMS>& c,
                                                       Lambda&& vec_field,
-                                                      const mfem::IntegrationRule& quad)
+                                                      const int npts)
 {
+#ifdef AXOM_USE_MFEM
+  static mfem::IntegrationRules my_IntRules(0, mfem::Quadrature1D::GaussLegendre);
+  QuadratureRuleView quad(my_IntRules.Get(mfem::Geometry::SEGMENT, 2 * npts - 1));
+#else
+  SLIC_WARNING_IF(npts > 100, "Axom's Gauss-Legendre nodes may be less precise for n > 100");
```

Comment:
Consider adding that configuring Axom w/ MFEM will provide more precise quadrature weights.

Reply:
I wonder if it's worthwhile to modify this warning to only print it out once. E.g if we call this on a loop w/ lots of BezierCurves, we'll get too many warnings about this.

(This might be better as a builtin SLIC macro -- `SLIC_*_ONCE` -- e.g. `SLIC_WARNING_IF_ONCE` )